### ISSUE TITLE
Add big users to promote project

### ISSUE THREAD

**Issue by ai** [lukeed/uvu#167]
We can mention PostCSS and Browserslist in docs to prove that this project was used by big tools and can be trusted

**Comment by byterose** [comment]
Good idea. Also [sveltekit](https://github.com/sveltejs/kit) is using uvu for testing.
